dist/build.gradle

```diff
@@ -197,3 +198,15 @@ task generateDependenciesReport(type: ConcatFilesTask) { concatDepsTask ->
 project.tasks.named('dependencyLicenses', DependencyLicensesTask) {
     it.dependencies = project.configurations.licenseChecks
 }
+
+
+tasks.register('copyPoms', Copy) {
```

Comment:
temporally we want to keep the build compatible with the existing DRA build. We can remove this workaround once we rely our releases on the new mechanism